resources/import.sql

```diff
@@ -78,11 +79,21 @@ CREATE TABLE IF NOT EXISTS npwd_sellout_listings (
   PRIMARY KEY (id)
 )
 ENGINE = INNODB,
-AUTO_INCREMENT = 26,
+AUTO_INCREMENT = 31,
```

Comment:
Doesn't really matter but whats the reasoning for this auto_increment number 🤔

Reply:
Just automated when I copied the whole thing kek. We should change that whole file to start on 0, right?